ajax/libs/gsap/1.18.0/easing/EasePack.min.js

```diff
@@ -0,0 +1,12 @@
+/*!
+ * VERSION: beta 1.15.2
```

Comment:
@jackdoyle Did you upload a testing version here? The cdnjs tag says `1.18.0` and this says `1.15.2` :)